r/README.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+# rrrow
+
+rrrow is an R front end to Apache Arrow. 
+
+## Installation
+
+I've only tested this locally for now, here is how I install it, borrowed
+from [rarrow](https://github.com/jimhester/rarrow). 
+
+First you need the Arrow C++ library installed
+
+```
+git clone https://github.com/apache/arrow.git
+cd arrow/cpp && mkdir release && cd release
+
+# It is important to statically link to boost libraries
+cmake .. -DCMAKE_BUILD_TYPE=Release -DARROW_BOOST_USE_SHARED:BOOL=Off
```

Comment:
This may vary a lot from system to system

Reply:
yes definitely. This is wip, I preferred having something that "worked for me" first to get started.